bgpd/bgp_encap.c

```diff
@@ -324,23 +324,7 @@ show_adj_route_encap (struct vty *vty, struct peer *peer, struct prefix_rd *prd)
   return CMD_SUCCESS;
 }
 
-enum bgp_show_type
```

Comment:
hello, I think the bgp_show_type should be moved to a single place.
I proposed it in https://github.com/freerangerouting/frr/pull/111 

If 113 does it, it is perfect, i wil adapt.